src/site/_includes/macros/authorcard.njk

```diff
@@ -0,0 +1,31 @@
+{% macro authorcard(collectionItem) %}
+
+{% set url = '/authors/'+collectionItem.key %}
```

Comment:
It looks like the old shortcode was using `collectionItem.url` for urls. Is that not possible here?